network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/figure_block2.html

```diff
@@ -5,9 +5,9 @@
 <div class="col-6 {% if count > 3 %}col-sm-3{% elif count == 3 %}col-sm-4{% elif count == 2 %}col-sm-6{% endif %} mb-2 mb-sm-4">
 	{% if url %}<a href="{{ url }}">{% endif %}
     {% if square %}
-      {% image value.image fill-890x890-c100 format-jpeg class="w-100" %}
+      {% image value.image fill-890x890-c100 format-jpeg class="w-100" alt=value.alt_text %}
```

Comment:
we'll need some fallbacks here

Reply:
Fair point. Will work on this shortly. Thanks!

Reply:
Added a StructValue to fallback the alt text to the image title in https://github.com/mozilla/foundation.mozilla.org/pull/5041/commits/652c1084b5c933e1fa5d3892b8402ca0f613736b